src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -242,6 +242,49 @@ def _parameters(registry_name,
     return parameters
 
 
+def get_credentials(cmd, auth_mode=None, credentials=None):
+    """Get the credential object from the input
+    :param str auth_mode: The login mode for the source registry
+    :param [] credentials: The list of credentials
+    """
+    # pylint: disable=line-too-long
+    Credentials, SourceRegistryCredentials, CustomRegistryCredentials, SourceRegistryLoginMode, SecretObject, SecretObjectType = cmd.get_models('Credentials', 'SourceRegistryCredentials', 'CustomRegistryCredentials', 'SourceRegistryLoginMode', 'SecretObject', 'SecretObjectType')
+
+    if auth_mode is None:
+        auth_mode = SourceRegistryLoginMode.default
```

Comment:
Do you want to check for `None` or empty here? This default should be at param level so it's visible to users. Also the server side enum is called `default`? What's the default behavior?